config/exchanges/CH_FR.yaml

```
@@ -1,3 +1,6 @@
+capacity:
+  - -2700
+  - 2700
```

```suggestion
capacity:
  - -2700
  - 1100
```
Should it not be the above? From the image you shared it looks like they have 1.1 GW import capacity and 2.7 export capacity?

(reversed here since we do CH to FR and not FR to CH)